backend/tests/api/endpoints/test_resource_library.py

```
@@ -0,0 +1,213 @@
+            ResourceMember.entity_id == str(consumer.id),
+            ResourceMember.status == MemberStatus.APPROVED,
+        )
+        .first()
+    )
+    copied_team = (
+        test_db.query(Kind)
+        .filter(
+            Kind.kind == "Team",
+            Kind.user_id == consumer.id,
+            Kind.name == source_team.name,
+            Kind.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)